src/gluonts/distribution/bijection.py

```diff
@@ -68,6 +68,10 @@ def inverse_bijection(self) -> 'Bijection':
     def event_dim(self) -> int:
         raise NotImplementedError()
 
+    @property
+    def sign(self) -> Tensor:
```

Comment:
Can you add a comment describing the contract of the method?

Reply:
done